test/tools/utils.ts

```diff
@@ -570,3 +570,41 @@ export function configureMongocryptdSpawnHooks(
     port
   };
 }
+
+/**
+ * Function for parsing an ISO 8601 formatted string into a Date object.
+ */
+export function toISODate(str: string | Date): Date {
```

Comment:
Is this functionally different than `new Date(Date.parse(str))`?

Reply:
It turns out this function is used just in one place, and I think it has no extra meaning, I just replaced it with `new Date(string)` in the test file and remove this function completly.